LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -291,18 +292,25 @@ - (void)handleLongPressGesture:(UILongPressGestureRecognizer *)gestureRecognizer
             
             self.currentView = [[UIView alloc] initWithFrame:collectionViewCell.frame];
             
-            collectionViewCell.highlighted = YES;
-            UIView *highlightedImageView = [collectionViewCell LX_snapshotView];
-            highlightedImageView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-            highlightedImageView.alpha = 1.0f;
+            BOOL cellHighlightedWhenPickedUp = collectionViewCell.highlighted;
+            UIView *highlightedImageView;
+            UIView *imageView;
```

Comment:
These two guys can move back inline and it should be a bit cleaner, `highlightedImageView` is only used in the conditional.